src/parser.js

```diff
@@ -267,8 +267,16 @@ export default class Parser extends Stream {
               this.manifest.discontinuityStarts.push(uris.length);
             },
             'program-date-time'() {
-              this.manifest.dateTimeString = entry.dateTimeString;
-              this.manifest.dateTimeObject = entry.dateTimeObject;
+              if (typeof this.manifest.dateTimeString === 'undefined') {
+                // PROGRAM-DATE-TIME is a media-segment tag, but for backwards
+                // compatibility, we add the first occurance of the PROGRAM-DATE-TIME tag
+                // to the manifest object
+                this.manifest.dateTimeString = entry.dateTimeString;
+                this.manifest.dateTimeObject = entry.dateTimeObject;
+              }
```

Comment:
You could still calculate stream time from currentTime by looking at the `dateTimeString` attached to the individual segment objects, the backwards compatibility is so updating to a version including this change wont break code that is expecting that information to be on the playlist object